crcmocks/entitlements.py

```diff
@@ -1,28 +1,40 @@
 from flask import jsonify
 from flask import Blueprint
+from flask import request
+
+from crcmocks.config import DEFAULT_SERVICES
+from crcmocks.util import get_user_rh_identity
 
 
 blueprint = Blueprint("entitlements", __name__)
 
 
-services = [
-    "ansible",
-    "cost_management",
-    "insights",
-    "advisor",
-    "migrations",
-    "openshift",
-    "settings",
-    "smart_management",
-    "subscriptions",
-    "user_preferences",
-]
-entitlements = {}
-for svc in services:
-    entitlements[svc] = {"is_entitled": True, "is_trial": False}
-
-
-# TODO: allow customizable entitlements response per user
+# TODO: support is_trial?
 @blueprint.route("/v1/services")
 def services():
-    return jsonify(entitlements)
+    # get the identity header
+    identity_header = request.headers.get("X-Rh-Identity")
+
+    if not identity_header:
+        return "No identity header present in request\n", 401
+
+    # get the user
+    user, username, account_number = get_user_rh_identity(identity_header)
+
+    if not user:
```

Comment:
Are we able to enhance these statements so we can precisely know what's not being found? It would help narrow down in the future if it's the username, or account number immediately causing the problem.

Reply:
Hmm, the DB query I make looks for either:
```
    return user_db.search(
        (query.account_number.search(str(account_number))) | (query.username == username)
    ), username, account_number
```
which I why I returned both. So with that being the case, if you hit this, it would mean that the neither the username nor the account_number were found in the DB.

Reply:
Ah, got it! 👍